flexget/components/notify/notifiers/mqtt.py

```diff
@@ -0,0 +1,185 @@
+                    client_key: /path/to/pem/encoded/client_certificate.key
+                    validate_broker_cert: True/False
+                    tls_version: ['tlsv1.2', 'tlsv1.1', 'tlsv1']
+                ]
+                [qos: [0,1,2] ]
+                [retain: True/False]
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'topic': {'type': 'string'},
+            'broker_address': {'type': 'string'},
+            'broker_port': {'type': 'integer', 'default': 1883},
+            'broker_timeout': {'type': 'integer', 'default': 30},
+            'broker_transport': {'type': 'string', 'default': 'tcp', 'enum': ['tcp', 'websockets']},
+            'broker_protocol': {'type': 'string', 'default': 'MQTTv311', 'enum': ['MQTTv31', 'MQTTv311']},
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'enable_encrypted_communication': {'type': 'boolean', 'default': False},
+            'certificates': {
+                'type': 'object',
+                'properties': {
+                    'broker_ca_cert': {'type': 'string'},
+                    'client_cert': {'type': 'string'},
+                    'client_key': {'type': 'string'},
+                    'validate_broker_cert': {'type': 'boolean', 'default': True},
+                    'tls_version': {'type': 'string', 'enum': ['tlsv1.2', 'tlsv1.1', 'tlsv1']},
+                },
+                'additionalProperties': False,
+            },
+            'qos': {'type': 'integer', 'minimum': 0, 'maximum': 2, 'default': 0},
+            "retain": {"type": "boolean", 'default': False},
+        },
+        'additionalProperties': False,
+        'required': ['broker_address','topic'],
+    }
+       
+    def notify(self, title, message, config):
+        """
+        Publish to an MQTT topic
+        """
+        try:
+            import paho.mqtt.client as mqtt
+
+        except ImportError as e:
+            logger.verbose('Error importing paho.mqtt.client: {}', e)
+            raise DependencyError(
+                plugin_name, 'paho.mqtt.client', 'paho-mqtt python module is required for MQTT notify plugin. ImportError: %s' % e
+            )
+
+        def on_log_cb(client, userdata, level, buff):
+            logger.verbose(str(buff))
+
+        def on_publish_cb(client, userdata, mid):
+            logger.verbose('MQTT on_publish callback -  message was successfully published to broker as messageID={}',str(mid))
+            client.disconnect()
+
+        def on_disconnect_cb(client, userdata, rc):
+            logger.verbose('MQTT on_disconnect callback - disconnected with result code {} [{}]',str(rc),conn_rc_description_map.get(rc),'Unknown')
+            client.loop_stop()
+            
+        config['title'] = title
+        config['message'] = message
+        config['payload'] = '{} - {}'.format(config.get('title'),config.get('message'))
```

Comment:
please use fstrings where applicable:
```suggestion
        config['payload'] = f"{config.get('title')} - {config.get('message')}"
```

Reply:
Fixed with commit ca2b843